mw-webapp/src/utils/auth/getNewUserCredentials.ts

```diff
@@ -0,0 +1,24 @@
+import {getRedirectResult} from "firebase/auth";
+import {NavigateFunction} from "react-router-dom";
+import {writeNewUserData} from "src/utils/auth/writeNewUserData";
+import {auth} from "src/firebase";
+
+export const getNewUserCredentials = async (navigateFunction: NavigateFunction, navigateDestination: string) => {
```

Comment:
this function violates single responsibility and it does not get anything. Refactor required

Reply:
Done